implementation/test/TBTCSystemTest.js

```diff
@@ -9,39 +9,48 @@ const {BN, expectRevert} = require("@openzeppelin/test-helpers")
 const {expect} = require("chai")
 
 const TBTCSystem = contract.fromArtifact("TBTCSystem")
+const TestTBTCDepositToken = contract.fromArtifact("TestTBTCDepositToken")
 
 describe("TBTCSystem", async function() {
   let tbtcSystem
   let ecdsaKeepFactory
+  let tdt
 
   before(async () => {
-    const {tbtcSystemStub, ecdsaKeepFactoryStub} = await deployAndLinkAll(
+    const {
+      tbtcSystemStub,
+      ecdsaKeepFactoryStub,
+      tbtcDepositToken,
+    } = await deployAndLinkAll(
       [],
       // Though deployTestDeposit deploys a TBTCSystemStub for us, we want to
       // test TBTCSystem itself.
-      {TBTCSystemStub: TBTCSystem},
+      {TBTCSystemStub: TBTCSystem, tbtcDepositToken: TestTBTCDepositToken},
     )
     // Refer to this correctly throughout the rest of the test.
     tbtcSystem = tbtcSystemStub
     ecdsaKeepFactory = ecdsaKeepFactoryStub
+    tdt = tbtcDepositToken
   })
 
   describe("requestNewKeep()", async () => {
     let openKeepFee
+    const keepOwner = accounts[2]
+
     before(async () => {
       openKeepFee = await ecdsaKeepFactory.openKeepFeeEstimate.call()
+      await tdt.forceMint(accounts[0], web3.utils.toBN(keepOwner))
```

Comment:
Can we extract `accounts[0]` to a constant as suggested in https://github.com/keep-network/tbtc/pull/514/files#r388833426. If you prefer we can name it `tdtOwner` instead of `depositOwner`, but let's make it different than `accounts[0]` which is a default account used by truffle.

We can use `tdtOwner` defined on a higher level to test if the request gets rejected when called by them as proposed in https://github.com/keep-network/tbtc/pull/514/files#r388835478

Reply:
we can name it, but the address here is not significant. we just need to mint the token and need to provide an address to do so.

Reply:
extracted TDT owner to accounts[1]